cartofacade/queue.go

```diff
@@ -0,0 +1,194 @@
+// Package cartofacade contains the api to call into CGO
+package cartofacade
```

Comment:
as much as possible, lets try to make anything in this package that can be private, private

if something is public only for testing lets call that out with a comment